cpp/src/phonenumbers/phonenumberutil.cc

```diff
@@ -227,7 +227,7 @@ string CreateExtnPattern(const string& single_extn_symbols) {
           "[:\\.．]?[  \\t,-]*", capturing_extn_digits, "#?|" */
       "[ \xC2\xA0\\t,]*(?:e?xt(?:ensi(?:o\xCC\x81?|\xC3\xB3))?n?|"
       "(?:\xEF\xBD\x85)?\xEF\xBD\x98\xEF\xBD\x94(?:\xEF\xBD\x8E)?|"
-      "доб|[", single_extn_symbols, "]|int|"
+      "\xD0\xB4\xD0\xBe\xD0\xB1|[", single_extn_symbols, "]|int|"
```

Comment:
Nit: `\xBE` (uppercase `E`) for consistency

Reply:
Done. Thanks for catching this :)